README.md

```diff
@@ -36,6 +36,15 @@ python -m venv r2ai
 . r2ai/bin/activate
 ```
 
+if you have this error, install this:
+The virtual environment was not created successfully because ensurepip is not
+available.  On Debian/Ubuntu systems, you need to install the python3-venv
+package using the following command.
+```
```

Comment:
I think that “python -v venv r2ai” should work without installing this package

Reply:
In Ubuntu 22.04.3 LTS .
I got some errors. If you want to add this item to the readme code, please.